packages/cli/test/unit/data/mocks/graph.frontmatter.js

```diff
@@ -0,0 +1,4 @@
+/* eslint-disable max-len */
+module.exports = {
```

Comment:
would it just make sense to update the existing mock graph instead of making this new one?

Reply:
I'd have to update every entry or replace it entirely. Too much changes.

Reply:
I would be fine with it in this case, if just to keep things simple to one file for the sack of keeping testing more streamlined.

I have it in our Trello board that in the next project we should just find a way to get a graph from the CLI somehow, so we can get ride of the files altogether.

Reply:
I ended up replacing it entirely and added [a quick piece of code](https://github.com/ProjectEvergreen/greenwood/blob/fa6a22b54bdabb6fca0770c1ad564081de3f12b3/packages/cli/src/lifecycles/graph.js#L182) you can uncomment if you ever need to output a new mock again.